e2e-tests/kuttl/scheduled-backup/05-update-cluster.yaml

```diff
@@ -0,0 +1,156 @@
+apiVersion: pxc.percona.com/v1-9-0
+kind: PerconaXtraDBCluster
```

Comment:
I see we apply backup schedule, but we don't stop it and test recovery - is it skipped on purpose?

Reply:
nope. I'm removing schedule section from shell script